Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/ContainerActionButtonHandler.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzFramework/DocumentPropertyEditor/PropertyEditorNodes.h>
+#include <AzToolsFramework/UI/DocumentPropertyEditor/PropertyHandlerWidget.h>
+#include <QToolButton>
+
+namespace AzToolsFramework
+{
+    class ContainerActionButtonHandler : public PropertyHandlerWidget<QToolButton>
+    {
+    public:
+        ContainerActionButtonHandler();
+
+        void SetValueFromDom(const AZ::Dom::Value& node);
+
+        static const AZStd::string_view GetHandlerName()
+        {
+            return AZ::DocumentPropertyEditor::Nodes::ContainerActionButton::Name;
+        }
```

Comment:
You can make this function constexpr since it is returning a string_view.

Reply:
Sure thing, done for this and the default impl of GetHandler